sln/OData.Pipeline.sln

```diff
@@ -0,0 +1,1484 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30011.22

```

Comment:
Part of the pipeline change is to start running `dotnet build` and `dotnet test` on the solution file instead of adding each `.csproj` to the yml. I would like for the pipelines to run with `OData.E2E.sln`; however, currently there are some projects that either don't build, or have failing tests:

1. microsoft.odata.service.design.t4.unittests
2. microsoft.odata.service.tddunittests
3. microsoft.test.odata.tests.client
4. datawebrules
5. serverwrapperapp

This new solution file is a copy of `OData.E2E.sln`, except without those 5 projects. The plan would be to fix those 5 projects over time and eventually remove `OData.Pipeline.sln` entirely, having the pipelines use `OData.E2E.sln` from that point forward.